vocabularies/Hierarchy.json

```diff
@@ -86,6 +86,31 @@
         "$Type": "Edm.PropertyPath",
         "$Nullable": true,
         "@Core.Description": "Integer property of type `Edm.Int64` holding the the number of [matching](#MatchCount) descendants a node has in the unlimited hierarchy"
+      },
+      "CreateAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that creates a new node, following [this template](#Template_CreateAction)"
+      },
+      "CreateChildAction": {
```

Comment:
And then the CreateAction is needed only for creating root nodes.

Reply:
No, it is also needed for creating child nodes in a separate request. Because the instance-bound action is meant for batch requests only: https://github.com/SAP/odata-vocabularies/blob/ba3cb87f7b7f8fcafd46a36864ca99ebd57021cf/vocabularies/Hierarchy.xml#L272-L277